deployments/container/Dockerfile

```diff
@@ -82,8 +89,9 @@ LABEL org.opencontainers.image.source="https://github.com/NVIDIA/k8s-dra-driver-
 # https://docs.docker.com/build/building/multi-platform/#install-qemu-manually
 RUN mkdir /licenses && mv /NGC-DL-CONTAINER-LICENSE /licenses/NGC-DL-CONTAINER-LICENSE
 
-COPY --from=build /artifacts/compute-domain-controller     /usr/bin/compute-domain-controller
-COPY --from=build /artifacts/compute-domain-kubelet-plugin /usr/bin/compute-domain-kubelet-plugin
-COPY --from=build /artifacts/compute-domain-daemon         /usr/bin/compute-domain-daemon
-COPY --from=build /artifacts/gpu-kubelet-plugin            /usr/bin/gpu-kubelet-plugin
-COPY --from=build /build/templates                         /templates
+COPY --from=toolkit /usr/bin/nvidia-ctk                      /usr/bin/nvidia-ctk
```

Comment:
My suggestion would be to use the `nvidia-cdi-hook` binary instead. From an `nvcdi` point of view these should be interchangeable.

Reply:
Changed in latest, PTAL